packages/kit/src/core/build/build_client.js

```diff
@@ -105,6 +107,12 @@ export async function build_client({
 	const entry_css = new Set();
 	find_deps(entry, vite_manifest, entry_js, entry_css);
 
+	fs.writeFileSync(
+		`${client_out_dir}/version.json`,
+		JSON.stringify(process.env.VITE_APP_VERSION),
```

Comment:
To make this future-proof it should be an object:

```suggestion
		JSON.stringify({ version: process.env.VITE_APP_VERSION })
```